lib/Parse/ParseDecl.cpp

```diff
@@ -2395,6 +2395,53 @@ bool Parser::parseTypeAttribute(TypeAttributes &Attributes, bool justChecking) {
                        diag::convention_attribute_expected_rparen,
                        LPLoc);
   }
+  
+  // SWIFT_ENABLE_TENSORFLOW
+  bool linear = false;
+  if (attr == TAK_differentiable) {
+    // Check if there is a 'linear' argument.
+    if (Tok.is(tok::l_paren) && peekToken().is(tok::identifier)) {
+      Parser::BacktrackingScope backtrack(*this);
+      consumeToken(tok::l_paren);
+      
+      // Determine if we have '@differentiable(linear) (T) -> U'
+      // or '@differentiable (linear) -> U'.
+      if (Tok.getText() == "linear" && consumeIf(tok::identifier)) {
+        
```

Comment:
```suggestion
```

Reply:
Will remove the extra line this time instead of just the whitespace 😉